frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -2,18 +2,18 @@
 Feature: Applications managed by GitOps and Create secret
     User should be able to see list of applications managed by GitOps
 
-
         Background:
-            Given user has installed gitOps operator
-              And user is at developer perspective
-
+            Given user has installed GitOps operator
+              And user has installed Pipelines operator
+              And user has installed Sealed Secrets operator in "cicd" namespace
+              And user has created Sealed Secrets instance of name "sealedsecretcontroller"
+              Then user is at developer perspective
```

Comment:
```suggestion
              And user is at developer perspective
```
In Background section, Only Given statements should be added

Reply:
Done